.github/workflows/ci.yml

```diff
@@ -7,12 +7,23 @@ on:
 
 jobs:
   basic:
+    strategy:
+      matrix:
+        image:
+          # Default fedora base image
+          - url: ""
+            base: ""
```

Comment:
We should be explicit here, even though the Fedora is default when nothing is specified, at first glance it's not clear which Fedora version is used.

Reply:
My intention here was to test that the previous default behavior (i.e. no env vars exported) keeps working. Would it help to put in more detailed comment?

Reply:
I see, in that case please add more detailed comment about the intention and that the default is `fedora:latest`.